bs/__tests__/tablecloth_test.ml

```diff
@@ -3,6 +3,13 @@ open Jest
 open Expect
 
 let () =
+  describe "Result" (fun () ->
+    describe "fromOption" (fun () ->
+      test "maps None into Error" (fun () -> expect Result.(fromOption "error message" None) |> toEqual (Belt.Result.Error "error message"));
```

Comment:
This should probably be
```suggestion
      test "maps None into Error" (fun () -> expect Result.(fromOption "error message" None) |> toEqual (Result.Error "error message"));
```
That it uses Belt under the covers is an implementation detail.